### DISCUSSION TITLE
Animate doesn't work

### DISCUSSION THREAD

**Discussion by svenbaus** [Megabit/Blazorise#3111] [Q&A] [answered]
I'm working on a prototype with blazorise and would like to use some animations. Coming from here (https://github.com/mikoskinen/Blazor.Animate) the framework says, it is hosted via Blazorise which is fine for me, since I like the extendability.

So I added the NuGet Package "Blazorise.Animate" and in my _Imports.razor:
`@using Blazorise.Animate `

I tried to get It working, but this doesn't come up animated:

```
<Blazorise.Animate.Animate Animation="Animations.SlideLeft" Duration="TimeSpan.FromSeconds(1)">
<Paragraph>TODO: Zweiter Schritt!</Paragraph>
</Blazorise.Animate.Animate>
```

So any ideas or documentation, on how to set up the library correctly?

Thanks :)

**Comment by stsrki** [comment, marked as answer]
You're probably missing `animate.js` in your index file. eg `Blazorise.Animate/blazorise.animate.js?v=0.9.4.4"></script>`

Check for example on https://github.com/Megabit/Blazorise/blob/rel-0.9.4/Demos/Blazorise.Demo.Bootstrap/wwwroot/index.html

**Reply by svenbaus** [reply]
Well, that was easy, thanks :)
